README.md

```diff
@@ -2,40 +2,94 @@
 
 ## About
 
-This repository contains our Docker-compose and setup bootstrap scripts used to create a deployment of the [UCSC Genomic Institute's](http://commons.ucsc-cgp.org) Computational Genomics Platform (CGP) for the NIH Data Commons Pilot on AWS. It uses, supports, and drives development of several key GA4GH APIs and open source projects. In many ways it is the generalization of the [PCAWG](https://dcc.icgc.org/pcawg) cloud infrastructure developed for that project and a potential reference implementation for the [NIH Commons](https://datascience.nih.gov/commons) concept.
+This repository contains our Docker-compose and setup bootstrap scripts
+used to create a deployment of the [UCSC Genomic
+Institute's](http://commons.ucsc-cgp.org) Computational Genomics
+Platform (CGP) for the NIH Data Commons Pilot on AWS. It uses, supports,
+and drives development of several key GA4GH APIs and open source
+projects. In many ways it is the generalization of the
+[PCAWG](https://dcc.icgc.org/pcawg) cloud infrastructure developed for
+that project and a potential reference implementation for the [NIH
+Commons](https://datascience.nih.gov/commons) concept.
 
 ## Components
-The system has components fulfilling a range of functions, all of which are open source and can be used independently or together. The installation instructions are currently specific to AWS, but usage of other cloud service providers is planned for the future.
+
+The system has components fulfilling a range of functions, all of which
+are open source and can be used independently or together. The
+installation instructions are currently specific to AWS, but usage of
+other cloud service providers is planned for the future.
 
 ![Computational Genomics Platform architecture](docs/dcc-arch.png)
 
-These components are setup with the install process available in this repository:
+These components are setup with the install process available in this
+repository:
 
-* [Boardwalk](boardwalk/README.md): our file browsing portal on top of Redwood
+* [Boardwalk](boardwalk/README.md): our file browsing portal on top of
+  Redwood
```

Comment:
I know this predates your changes (it was something that got overlooked), but can you instead say something like `...on top of the cgp-data-store`. Or maybe `on top of the Azul indexer`, as the UI doesn't directly interact with the data store.

Reply:
Done.